seed/challenges/02-javascript-algorithms-and-data-structures/es6.json

```diff
@@ -225,32 +225,25 @@
     },
     {
       "id": "587d7b87367417b2b2512b43",
-      "title": "Arrow functions",
+      "title": "Write Arrow Functions",
       "description": [
-        "In JS, we often, we often don't name our functions. Callback functions, for example.",
-        "Because we often create function just to pass it as argument to some other function, and we might not need to reuse this function anywhere.",
-        "To achieve this, we often use the boilerplate",
-        "<code>function(){</code>",
-        "<code>// function body</code>",
-        "<code> return ;</code>",
-        "<code>}</code>",
-        "ES6 provides you with the syntactic sugar to not having to write this. Instead, you can use arrow function syntax",
-        "<code>() => (</code>",
-        "<code>// function body</code>",
-        "<code>return ;</code>",
-        "<code>)</code>",
-        "When there is no function body, and only a return value, you can write that as a one-liner:",
-        "<code>() => (/* return value */)</code>",
-        "You don't have to write the keyword return here. You can even omit the paranthesis after =>",
-        "Instructions.",
-        "Refactor the function which returns the new Date(), to use arrow function syntax. Also make sure var is not there in the code anymore.",
+        "In JavaScript, we often don't need to name our functions, especially when passing a function as an argument to another function. Instead, we create inline functions. We don't need to name these functions because we do not reuse them anywhere else.",
+        "To achieve this, we often use the following syntax:",
+        "<blockquote>const myFunc = function() {<br>  const myVar = \"value\";<br>  return myVar;<br>}</blockquote>",
+        "ES6 provides us with the syntactic sugar to not have to write anonymous functions this way. Instead, you can use <strong>arrow function syntax</strong>:",
+        "<blockquote>const myFunc = () => {<br>  const myVar = \"value\";<br>  return myVar;<br>}</blockquote>",
+        "When there is no function body, and only a return value, arrow function syntax allows you to omit the keyword <code>return</code> as well as the brackets surrounding the code. This helps simplify smaller functions into one-line statements:",
+        "<blockquote>const myFunc= () => \"value\"</blockquote>",
+        "This code will still return <code>value</code> by default.",
+        "<h4>Instructions</h4>",
```

Comment:
Not related to your PR but since you are at it, Can you please replace this with a `<hr>` instead?